audify/cli.py

```
@@ -2,36 +2,532 @@
+        click.echo("--------\t----")
+        for lang, code in AVAILABLE_LANGUAGES.items():
+            click.echo(f"{lang:<10}\t{code}")
+        click.echo("=" * terminal_width)
+        return
+
+    if list_models:
+        click.echo("=" * terminal_width)
+        click.echo("Available models:".center(terminal_width))
+        click.echo("=" * terminal_width)
+        try:
+            response = requests.get(f"{KOKORO_API_BASE_URL}/models")
```

Ruff (0.15.9)

[error] Probable use of `requests` call without timeout

(S113)